holidays/countries/nepal.py

```diff
@@ -0,0 +1,366 @@
+        )
+        StaticHolidays.__init__(self, cls=NepalStaticHolidays)
+        super().__init__(*args, **kwargs)
+
+    def _add_non_continuous_holidays(self, is_workday: bool = False):
+        """Holidays removed by MoHA between 2019-2020."""
+
+        # https://www.bolpatra.gov.np/egp/openDateConverter
+        martyrs_day_dates = {
+            2010: (JAN, 30),
+            2011: (JAN, 30),
+            2012: (JAN, 30),
+            2013: (JAN, 29),
+            2014: (JAN, 30),
+            2015: (JAN, 30),
+            2016: (JAN, 30),
+            2017: (JAN, 29),
+            2018: (JAN, 30),
+            2019: (JAN, 30),
+            2020: (JAN, 30),
+            2021: (JAN, 29),
+            2022: (JAN, 30),
+            2023: (JAN, 30),
+            2024: (JAN, 30),
+            2025: (JAN, 29),
+            2026: (JAN, 29),
+            2027: (JAN, 29),
+            2028: (JAN, 29),
+            2029: (JAN, 29),
+            2030: (JAN, 29),
+            2031: (JAN, 30),
+            2032: (JAN, 30),
+            2033: (JAN, 29),
+        }
+
+        if (2019 <= self._year <= 2020) == is_workday:
+            if self._year in martyrs_day_dates:
+                # Martyr's Day.
+                self._add_holiday("Martyr's Day", martyrs_day_dates[self._year])
+
+            # National Democracy Day.
+            self._add_holiday_feb_19("National Democracy Day")
+
+            # Republic Day.
+            self._add_holiday_may_29("Republic Day")
+
+            # Hindu Holidays.
+
+            # Ram Navami.
+            self._add_ram_navami("Ram Navami")
+
+            # Janai Purnima.
+            self._add_raksha_bandhan("Janai Purnima")
+
+            # Shree Krishna Janmashtami.
+            self._add_janmashtami("Shree Krishna Janmashtami")
+
+            # Ghatasthapana.
+            self._add_sharad_navratri("Ghatasthapana")
+
+            # Duwadashi (Dashain).
+            self._add_papankusha_duwadashi("Duwadashi (Dashain)")
+
+    def _populate_public_holidays(self):
+        if self._year >= 2023:
+            # Prithvi Jayanti.
+            self._add_holiday_jan_11("Prithvi Jayanti")
+
+        # Women's Day.
+        self._add_womens_day("Women's Day")
+
+        # Nepal New Year.
+        self._add_vaisakhi("Nepali New Year")
+
+        # Labor Day.
+        self._add_labor_day("Labor Day")
+
+        # Constitution Day.
+        self._add_holiday_sep_19("Constitution Day")
+
+        # Christmas Day.
+        self._add_christmas_day("Christmas Day")
+
+        # Tamu Losar.
+        self._add_holiday_dec_30("Tamu Losar")
+
+        # Hindu holidays.
+
+        # Maghe Sankranti.
+        self._add_makar_sankranti("Maghe Sankranti")
+
+        # Sonam Losar.
+        self._add_sonam_losar("Sonam Losar")
+
+        # Gyalpo Losar.
+        self._add_gyalpo_losar("Gyalpo Losar")
+
+        # Maha Shivaratri.
+        self._add_maha_shivaratri("Maha Shivaratri")
```

Comment:
```suggestion
        # Maha Shivaratri.
        self._add_maha_shivaratri("Maha Shivaratri")

        # Gyalpo Losar.
        self._add_gyalpo_losar("Gyalpo Losar")
```
Chronological order.